src/Action/NullAction.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ApiPlatform\Core\Action;
+
+/**
+ * Empty action. Useful to trigger the kernel.view event without doing anything specific in the action
```

Comment:
It allows to create custom operations triggering the event system with only config. It's why I've renamed it. Why not doing it now?